client/src/App.js

```diff
@@ -329,6 +340,18 @@ class App extends Component {
                 <Segment>
                   Count in store is {this.props.count ? this.props.count : 0}
                 </Segment>
+                <SearchablePooledMultiSelectDropdown
```

Comment:
This is for demo in the `playground` route, which will not be present in release branch